src/Microsoft.AspNet.Http.Abstractions/HttpContext.cs

```diff
@@ -36,6 +36,8 @@ public abstract class HttpContext : IDisposable
 
         public abstract ISession Session { get; set; }
 
+        public abstract string TraceIdentifier { get; set; }
```

Comment:
API change. I actually think this should be exposed but this change just went from being an optimization to an API change.